### ISSUE TITLE
Contact Us button not visible in Light Mode

### ISSUE THREAD

**Issue by Shreyaraut12** [Mohitjadaun2026/PMS-CGC-U#85]
Hey @Poushmita ,
In light mode, the Contact Us button is not visible. This creates usability and accessibility issues since users are unable to locate or interact with the button properly.
<img width="943" height="258" alt="Image" src="https://github.com/user-attachments/assets/e59e3eb5-9734-471c-8000-577c74bff393" />
please assign me this issue!

**Comment by 213sanjana** [comment]
@Shreyaraut12 Assigned

**Comment by 213sanjana** [comment]
> [@213sanjana](https://github.com/213sanjana) This issue was already done What was it assigned to me I did the changes !

I understand but the issue has been assigned to somebody else.
You can contribute to other issues or raise one of your own
